rules/aws/initial_access_root_console_failure_brute_force.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2020/07/21"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an abnormal number of failed attempts to login to the AWS management console as the Root user identity. An
+adversary may attempt to brute force the Root credentials, as the Root account will have complete access to all AWS
+services and resources.
+"""
+from = "now-20m"
+index = ["filebeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "AWS Management Console Attempted Root Login Brute Force"
+note = "The AWS Filebeat module must be enabled to use this rule."
+references = ["https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html"]
+risk_score = 73
+rule_id = "4d50a94f-2844-43fa-8395-6afbd5e1c5ef"
+severity = "high"
+tags = ["AWS", "Elastic"]
+type = "threshold"
+
+query = '''
+event.module:aws and event.dataset:aws.cloudtrail and event.provider:signin.amazonaws.com and event.action:ConsoleLogin and aws.cloudtrail.user_identity.type:Root and event.outcome:failure
+'''
+
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1110"
+name = "Brute Force"
+reference = "https://attack.mitre.org/techniques/T1110/"
+
+
+[rule.threat.tactic]
+id = "TA0006"
+name = "Credential Access"
+reference = "https://attack.mitre.org/tactics/TA0006/"
+
+[rule.threshold]
+field = ""
+value = 5
```

Comment:
5 Failures in 5 minutes could fall in the realm of user error - I would expect brute force to be a smaller window with maybe more failures. Thoughts on tweak threshold value and interval?

Reply:
I was open to changing the threshold. But really, the Root user identity should never be used. Instead, an IAM user should be used to manage tasks. Any failed login from the Root may be anomalous, I just wanted to bump it up a bit more than just one failure. I'm open to bumping it up a bit more though.